server/src/main/java/io/crate/expression/udf/UserDefinedFunctionService.java

```diff
@@ -301,7 +301,11 @@ void validateFunctionIsNotInUseByGeneratedColumn(String schema,
 
         for (var indexParts : indices) {
             var tableInfo = docTableFactory.create(indexParts.toRelationName(), currentState);
-            TableReferenceResolver tableReferenceResolver = new TableReferenceResolver(tableInfo.columns(), tableInfo.ident());
+            var functionParameters = tableInfo.generatedColumns().stream()
```

Comment:
Not a big fan of the streams usage, I'd personally convert to a for loop, but also not strong opinion.

Reply:
Also done, Thanks!